app/views/home.js

```diff
@@ -1,34 +1,8 @@
 "use strict";
 
-
-
-//  P A C K A G E
```

Comment:
Why break convention in a repo? Like I mentioned elsewhere, being able to glance and see where you need to go is super helpful.

Reply:
every JS file has import or require statements at the top, it didn't seem necessary to have a comment explaining that that's what they are